experimental/packages/opentelemetry-exporter-metrics-otlp-grpc/src/OTLPMetricExporter.ts

```diff
@@ -66,12 +66,8 @@ class OTLPMetricExporterProxy extends OTLPGRPCExporterNodeBase<ResourceMetrics,
           : DEFAULT_COLLECTOR_URL;
   }
 
-  convert(metrics: ResourceMetrics[]): otlpTypes.opentelemetryProto.collector.metrics.v1.ExportMetricsServiceRequest {
-    return toOTLPExportMetricServiceRequest(
-      metrics[0],
-      this._aggregationTemporality,
-      this
-    );
+  convert(metrics: ResourceMetrics[]): IExportMetricsServiceRequest {
+    return createExportMetricsServiceRequest(metrics[0], this._aggregationTemporality);
```

Comment:
What if instead of using `metrics[0]` here `metrics` would be mapped over instead? And then combine the `resourceMetrics` fields into a single field?